src/editors/playgroundController.ts

```diff
@@ -581,8 +581,6 @@ export default class PlaygroundController {
 
     this._playgroundResult = evaluateResponse.result;
 
-    this._explorerController.refresh();
```

Comment:
This triggers the click event on the selected in the tree view document. We do not refresh lists on the fly on other explorers, so let's also use the existing refresh button in this case.